src/rest/v1/index.ts

```diff
@@ -75,3 +77,17 @@ v1.get(shiftsBase, async (req, res) => {
   const shifts = await listAllShifts();
   res.send(JSON.stringify({ shifts }));
 });
+
+// Ratings
+v1.post("/rate-worker", async (req, res) => {
+  const { workerUuid, shiftUuid, rating } = req.body;
+  const shiftAssignment = await rateWorker(workerUuid, shiftUuid, rating);
+  res.send(JSON.stringify({ shiftAssignment }));
+});
+
+// Block workers
+v1.post("/block-worker", async (req, res) => {
```

Comment:
What about API for unblocking worker ? If this is missing clarify with business if this is needed

Reply:
Yeah...that's a good idea.